### ISSUE TITLE
Breakpoint set a wrong line number when resolving source map

### ISSUE THREAD

**Issue by auchenberg** [Microsoft/vscode-chrome-debug#162]
**Actual:**
1. Set breakpoint at line number `200` in `app/controllers/app.js`
2) Breakpoint set at line number **`234`** within `app/controllers/app.js` open in VS Code.

**Expected:**
1. Set breakpoint at line number `200` in `app/controllers/app.js`
2) Breakpoint set at line number **`200`** within `app/controllers/app.js` open in VS Code.

**Cause:**
Mapped line number from sourcemapped file `app_bundled.js` is used.

**Log:**
```json
›From client: setBreakpoints({"source":{"path":"/Users/auchenberg/Development/timey.in/app/controllers/app.js"},"lines":[200],"breakpoints":[{"line":200}]})
›SourceMaps.setBP: Mapped /Users/auchenberg/Development/timey.in/app/controllers/app.js to http://localhost:8080/app_bundled.js
›SourceMaps.setBP: Mapped /Users/auchenberg/Development/timey.in/app/controllers/app.js:199:0 to http://localhost:8080/app_bundled.js:233:0
›Paths.setBP: http://localhost:8080/app_bundled.js is already a URL
›To target: {"id":26,"method":"Debugger.setBreakpointByUrl","params":{"url":"http://localhost:8080/app_bundled.js","lineNumber":233,"columnNumber":0}}
›From target: {"id":26,"result":{"breakpointId":"http://localhost:8080/app_bundled.js:233:0","locations":[{"scriptId":"38","lineNumber":233,"columnNumber":4}]}}
›SourceMaps.setBP: Can't map http://localhost:8080/app_bundled.js:233:4, keeping the line number as-is.
›To client: {"seq":0,"type":"response","request_seq":22,"command":"setBreakpoints","success":true,"body":{"breakpoints":[{"verified":true,"line":234}]}}
````

**Comment by auchenberg** [comment]
Solved by downgrading to version 0.2.5, so it must be the refactor of the sourcemap logic that causes something to go wrong.

**Comment by roblourens** [comment]
This issue was moved to Microsoft/vscode-chrome-debug-core#24
